tools/c7n_tencentcloud/c7n_tencentcloud/client.py

```diff
@@ -0,0 +1,55 @@
+# SPDX-License-Identifier: Apache-2.0
+
+import os
+from tencentcloud.common.credential import Credential
+from tencentcloud.common.profile.client_profile import ClientProfile
+from tencentcloud.common.profile.http_profile import HttpProfile
+from tencentcloud.common.common_client import CommonClient
+
+
+ENV_AK = "TENCENTCLOUD_SECRET_ID"
+ENV_SK = "TENCENTCLOUD_SECRET_KEY"
+
+
+def get_aksk():
```

Comment:
aksk is little idiosyncratic terminology, perhaps just rename this to get_credentials and have it return Credentials. also if credentials are missing in the environment, it would be nice to fail with an appropriate error message (MissingCredentials) instead of a generic key error.

Reply:
We have updated this by using the method in SDK, which support 3 ways to create credential.
commit: [e7ff155](https://github.com/cloud-custodian/cloud-custodian/pull/7688/commits/e7ff1551bc93a998fb3b23d3808042eaac9d976d)